src/plone/volto/patches.py

```diff
@@ -16,14 +16,10 @@
 
 LOG = logging.getLogger("Zope.SiteErrorLog")
 
-
-try:
-    from collective.folderishtypes.dx import content  # noqa F401
```

Comment:
This breaks compatibility with collective.folderishtypes, if anyone is still using it, and needs to be mentioned as a breaking change in the release notes.